corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -20,6 +22,7 @@
 from casexml.apps.phone.restore import CachedResponse, RestoreConfig, RestoreParams, RestoreCacheSettings
 from casexml.apps.case.xml import V2, V1
 from casexml.apps.case.sharedmodels import CommCareCaseIndex
+from casexml.apps.case.const import CASE_INDEX_EXTENSION
```

Comment:
Should we synchronize on our test plans? the one at

https://github.com/dimagi/commcare/blob/master/tests/resources/case_relationship_tests.json

is up to date with our fixes.

On Fri, Nov 6, 2015 at 4:48 PM, dimagimon <notifications@github.com> wrote:

> In corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py
> <https://github.com/dimagi/commcare-hq/pull/9012#discussion_r44191422>:
>
> > @@ -20,6 +22,7 @@
> >  from casexml.apps.phone.restore import CachedResponse, RestoreConfig, RestoreParams, RestoreCacheSettings
> >  from casexml.apps.case.xml import V2, V1
> >  from casexml.apps.case.sharedmodels import CommCareCaseIndex
> > +from casexml.apps.case.const import CASE_INDEX_EXTENSION
>
> F401 'CASE_INDEX_EXTENSION' imported but unused
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9012/files#r44191422>.
>